src/components/AvatarUploadModal.js

```diff
@@ -0,0 +1,75 @@
+import BasicModal from "./BasicModal";
+import ProfileIcon from "./icons/ProfileIcon";
+import { Alert, DialogActions, Typography } from "@mui/material";
+import { useState } from "react";
+import api from "../capableApi/index";
+import { useCurrentPatient } from "../fetchDataHooks";
+import * as Sentry from "@sentry/react";
+
+export default function AvatarUploadModal(props) {
+  const { currentPatient } = useCurrentPatient();
+  const [selectedImage, updateSelectedImage] = useState(null);
+  const onFileChange = event => {
+  updateSelectedImage(event.target.files[0]);
+  }
+  const [failedRequest, updateFailedRequest] = useState(false);
```

Comment:
With more time, I would like to be using react-query to make this request which would automatically provide the error state. But if we're in a crunch, this is fine for now. Since it looks like this component isn't being used in this PR anyway, you could separate it out to its own PR and we could maybe address the react-query thing there. Otherwise if keeping the `useState`, would make the following suggestion
```suggestion
  const [failedRequest, setFailedRequest] = useState(false);
```